pkg/controllers/work/apply_controller_test.go

```diff
@@ -80,6 +80,7 @@ var (
 	testManifest         = workv1alpha1.Manifest{RawExtension: runtime.RawExtension{
 		Raw: rawTestDeployment,
 	}}
+	largeSecret v1.Secret
```

Comment:
please declare the variable close to where you use it to improve the readability.